packages/admin/admin/src/index.ts

```diff
@@ -196,3 +196,5 @@ export { RouterTabsClassKey } from "./tabs/RouterTabs.styles";
 export { Tab, Tabs, TabsProps } from "./tabs/Tabs";
 export { TabsClassKey } from "./tabs/Tabs.styles";
 export { TabScrollButton, TabScrollButtonClassKey, TabScrollButtonProps } from "./tabs/TabScrollButton";
+export { TranslationConfigProvider } from "./translator/config/TranslationProvider";
+export { useTranslationConfig } from "./translator/config/useTranslationConfig";
```

Comment:
I'm not sure about the naming... Wouldn't `TranslationServiceProvider` and `useTranslationService` be a better fit?

Reply:
@thomasdax98 @nsams what do you think?

Reply:
I also like `TranslationServiceProvider` and `useTranslationService` better

Reply:
We should even consider prefixing it with "Content", `ContentTranslationServiceProvider` and `useContentTranslationServiceProvider` 🤔